src/content/NewsContent.tsx

```diff
@@ -66,55 +66,55 @@ const news: PostDescription[] = [
   },
   {
     title: 'June 2022 News',
-    text: 'Learn about the events we attended and the enhancements to the BPMN Visualization TS library that took place in June 2022.',
+    text: 'Learn about the events we attended and the enhancements to the bpmn-visualization TypeScript library that took place in June 2022.',
```

Comment:
As for articles, we shouldn't update past news afterwards. See the comment about the same changes in the `NewsContent.tsx` file. 

This comment applies to all related changes currently proposed in this file.

Reply:
I change it yesterday on the articles to keep consitency between all the artciles, as we didn't change the name of the library.